spec/features/support/sign_up_helpers.rb

```diff
@@ -13,13 +15,11 @@ def create_password_for_account(password: 'supersecret', confirmed_password: 'su
 end
 
 def confirmation_email_link
-  confirmation_email = ActionMailer::Base.deliveries.first
-  parsed_email = Nokogiri::HTML(confirmation_email.body.to_s)
-  parsed_email.css('a').first['href']
+  ConfirmationUseCaseSpy.last_confirmation_url
 end
 
 def confirmation_email_received?
-  ActionMailer::Base.deliveries.any?
+  !ConfirmationUseCaseSpy.last_confirmation_url.nil?
```

Comment:
Could just use .present? here

Reply:
This way it will work in specs which not load `rails_helper`